server/middleware/setUpWebSecurity.ts

```diff
@@ -22,6 +22,7 @@ export default function setUpWebSecurity(): Router {
   // page by an attacker.
 
   const scriptSrc = ["'self'", (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`]
+  const connectSrc = ["'self' *.applicationinsights.azure.com"]
```

Comment:
This gives the browser permission to send telemetry data to the app insights domain.